libs/vre/shared/app-resource-page/src/lib/resource-representation.component.ts

```diff
@@ -16,19 +16,20 @@ import { filter, map } from 'rxjs/operators';
 @Component({
   selector: 'app-resource-representation',
   template: ` <div class="representation-container center" [ngSwitch]="representationToDisplay.fileValue.type">
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      [attachedProject]="attachedProject$ | async"
-      *ngSwitchCase="representationConstants.stillImage"
-      [resource]="resource.res" />
-    <app-still-image
-      #stillImage
-      class="dsp-representation stillimage"
-      *ngSwitchCase="representationConstants.externalStillImage"
-      [attachedProject]="attachedProject$ | async"
-      [resource]="resource.res" />
-
+    <ng-container *ngIf="attachedProject$ | async as attachedProject">
```

Comment:
why are there two still images components ?

Reply:
for me it also looks weird, but it is already something about refactoring, not related with this fix

Reply:
@domsteinbach I am removing the second instance of the still image component here. Please let me know if it is required for some reason.